core/platform/protobuf/gce_protobuf_services_test.py

```diff
@@ -0,0 +1,56 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for gce metadata services"""
+
+from core.platform.protobuf import gce_protobuf_services
+from core.platform.protobuf.protofiles import text_classifier_pb2 # pylint: disable=no-name-in-module
+from core.tests import test_utils
+import vmconf
+
+class TrainingJobDataResultTests(test_utils.GenericTestBase):
+    """Tests for TrainingJobResultMessage class."""
+
+    def test_that_all_algorithm_ids_have_mapping(self):
+        """Test that TrainingJobDataMessage contains mapping for all
+        algorithm ids.
+        """
+        # Get value of 'param' parameter.
```

Comment:
I don't see the use of "param" anywhere?

Reply:
Removed.